module/VuFind/tests/integration-tests/src/VuFindTest/View/Helper/Root/ComponentTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * Component Test Class
+ *
+ * PHP version 7
```

Comment:
When we update this PR, we need to remember to change all PHP version 7 comments to PHP version 8 to reflect our raised requirements.